crates/shared/src/tenderly_api.rs

```diff
@@ -169,7 +169,7 @@ pub struct SimulationRequest {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
-#[serde(rename_all = "lowercase")]
```

Comment:
We don't control the API, so this may break.

Reply:
Reverted